### ISSUE TITLE
Working image/video support

### ISSUE THREAD

**Issue by IoIxDButStronger** [robinuniverse/TwitFix#3]
I know that the focus is gif embeds, but having a way to link to tweets with image tweets in a compact matter similar to this would be nice. 

Video tweets seem to work on desktop but on the iOS Discord app (which I am currently being forced to use, otherwise I would normally use Android) videos don't seem to play, and a video length of 0:00 is reported. This might be a bug with Discord but I'm curious to know if it's something that can be fixed through the site's end.

**Comment by robinuniverse** [comment]
I will try to add more info to the embed, but I don't have any iOS devices to test this with, and to be honest mobile stuff is kinda out of scope, though I would like to see about doing some interesting image stuff later on

**Comment by adryd325** [comment]
I'm unable to reproduce your videos issue on the Discord iOS app (83.0).

**Comment by IoIxD** [comment]
What is your iOS version? I'm on 14.6.

**Comment by adryd325** [comment]
15.0

**Comment by robinuniverse** [comment]
Yeah, actually, I'm not sure what this is about because I borrowed a friends iPhone out of curiosity and get no issues on a iOS 14.6 iPhone 7, works better than it does on android, actually, can anyone else reproduce?

**Comment by robinuniverse** [comment]
Actually, scratch that, it's some videos that work and some videos don't - I have a hunch about why this is, and it's a little inconvenient to fix if it is the case, but basically, every tweet has a few different versions of a vid link (usually just different resolutions and bitrates) but sometimes they will have a file formatted as m3u8 instead of MP4, I take very minimal steps to prefer the MP4 version every time, but I'm by no means guaranteeing that m3u8 files are never embedded, I have a feeling that those are the files that are having problems, this is just a hunch though, I'll need to do some digging around to figure out for sure

**Comment by mina9999** [comment]
the image embed is on thumbnail not image so its small.
